docs/transparent-validation/rte/v1.1.0/quick-start-guide.md

```diff
@@ -72,3 +72,27 @@ common preparation of RTE without listed functions.
 1. Login into the device by using the default credentials:
     - Login: `root`
     - Password: `meta-rte`
+
+## Basic power operations
+
+Basic power operations should be performed based on the `rte_ctrl` script
+implemented in `meta-rte` (OS image dedicated to the RTE platform). To perform
+basic power operations use the below-described commands:
+
+1. Turn on the platform:
```

Comment:
The commands below are only correct for a specific OS version. Please add an appropriate note.

Reply:
Updated